--- a/Makefile	2019-09-21 14:07:05.601708172 +0300
+++ b/Makefile	2019-09-21 20:06:27.115176828 +0300
@@ -22,7 +22,7 @@
 
 install: all
 	rm othman-data/quran-kareem.png || :
-	$(PYTHON) setup.py install -O2 --root $(DESTDIR)
+	$(PYTHON) setup.py install --prefix=/app
 	$(INSTALL) -d $(datadir)/applications/
 	$(INSTALL) -m 0644 Othman.desktop $(datadir)/applications/
 	for i in 96 72 64 48 36 32 24 22 16; do \
